localization/ekf_localizer/src/check.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ekf_localizer/check.hpp"
+
+#include "ekf_localizer/warning.hpp"
+
+#include <fmt/core.h>
+
+#include <string>
+
+bool checkDelayStep(const Warning & warning, const int delay_step, const int extend_state_step)
```

Comment:
How about something like this?
```suggestion
bool isDelayStepBelowThreshold(const Warning & warning, const int delay_step, const int delay_step_threshold)
```

`delay_step_threshold` can also be something like `maximum_state_step` or `maximum_delay_step`.

Reply:
Updated [38426bc](https://github.com/autowarefoundation/autoware.universe/pull/2657/commits/38426bc828f44a8619954193b7448b49dbc08b67), [296b6bb](https://github.com/autowarefoundation/autoware.universe/pull/2657/commits/296b6bb403b0c528e061af59d792e7800672210c)

Reply:
Thank you!